src/components/user/tour/tour.js

```diff
@@ -1,6 +1,26 @@
 import React from "react";
-import Typography from "@material-ui/core/Typography";
+import { DataGrid } from '@material-ui/data-grid';
+import { TOUR_ROWS, TOUR_COLUMNS } from './utils/constants';
 
-export default function Tour() {
-  return <Typography paragraph>The 'Tour' Page comming soon...</Typography>;
-}
+export default class Tour extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      tourData: []
+    };
+  }
+
+  getData(){
+    this.setState({tourData: TOUR_ROWS});
+  };
+
+  componentDidMount () {
+    this.getData();
+  };
+
+  render() {
+    return (
+        <DataGrid rows={this.state.tourData} columns={TOUR_COLUMNS} autoHeight='true' autoPageSize='true' />
```

Comment:
the last thing :) 

[https://material-ui.com/api/data-grid/#props](https://material-ui.com/api/data-grid/#props )
Here you can see, that type of `autoHeight` and `autoPageSize` props is boolean, so you need to pass the boolean values, and not a string. So it will be `autoHeight={true}` instead of `autoHeight='true'`. 
Also for boolean props, you can use the shorthand version. 
`<DataGrid autoHeight={true}>...` and `<DataGrid autoHeight >...` are the same.

Reply:
ready

Reply:
ready